src/browser/provider/built-in/phantomjs.js

```diff
@@ -0,0 +1,47 @@
+import { create as createPhantom } from 'phantom';
+
+export default {
+    phantom: null,
+
+    openedPages: {},
```

Comment:
What's wrong with `listAvailableBrowserNames` or `listAvailableBrowsers` ? Why is that `optional`?